packages/guides-restructured-text/src/RestructuredText/Parser/InlineLexer.php

```diff
@@ -60,10 +60,9 @@ protected function getCatchablePatterns(): array
             '(?<=^|\s)[a-z0-9-]+_{2}', //Inline href.
             '(?<=^|\s)[a-z0-9-]+_{1}(?=[\s\.+]|$)', //Inline href.
             '``.+?``(?!`)',
+            '<[^<]+?>(?=`)',
```

Comment:
This will catch the same pattern within normal texts. The normal text will then be treated as one token missing other things that might be inside. What will for example happen to the following: 

:code:`<` and then :code:`>` Will this give us a false positive? It might. At least add tests for it?